web/themes/custom/usagov/sass/_benefits-block.scss

```diff
@@ -82,6 +82,7 @@ a.usa-button.benefits-link {
   font-size: 22px;
   font-weight: 600;
   width: 100%;
+  border: solid 2px transparent;
```

Comment:
This was added due to a bug i found... when reducing the the size of the icon the border that is getting applied on hover was causing the block to shift. this will help address the issue